src/program/lwaftr/doc/README.continuous-integration.md

```diff
@@ -0,0 +1,60 @@
+# Continuous integration
+
+The Snabb project has a continuous integration lab, described in [the docs]
```

Comment:
Maybe "The snabblab docs" or something? "The docs" is really uninformative as a link.